generate_en_nodes.py

```diff
@@ -0,0 +1,38 @@
+"""
+generate_en_nodes
+
+Create en_nodes.json file to be used in dubbed video mappings.
+
+Usage:
+  generate_en_nodes.py
+"""
+
+import json
+import shutil
+import os
+
+from collections import OrderedDict
+from contentpacks.khanacademy import retrieve_kalite_data
+
+PROJECT_PATH = os.path.realpath(os.path.dirname(os.path.realpath(__file__)))
+EN_NODE_PATH = os.path.join(PROJECT_PATH, "build", 'en_nodes.json')
+RESOURCES_PATH = os.path.join(PROJECT_PATH, "contentpacks", "resources")
+
+
+def main():
+    """
+    This will generate en_nodes.json file that will be used in dubbed video mappings.
+    """
+    lang = "en"
+    name = "en_nodes.json"
```

Comment:
I will remove generate_en_nodes.py and move it before running the dubbed video mappings. And also the generate_dubbed_video_mappings.py will be move on to contentpacks module because it's not dependent.